### DISCUSSION TITLE
Does dragToSeek work if i use custom peaks data?

### DISCUSSION THREAD

**Discussion by Miltonbhowmick** [katspaugh/wavesurfer.js#3838] [Q&A]
Here is an example of custom peaks using via Wavesurfer,
[https://wavesurfer.xyz/examples/?predecoded.js](url)

In the above example, i am really confused on dragToSeek feature. When i click on a specific portion of the wave, the audio does not move to that position and continue playing.

My Try:
```
wavesurferInstances.current = musicTrackList.map((music, index) => {
        const ws = WaveSurfer.create({
          container: waveformRefs.current[index],
          waveColor: "#828282",
          progressColor: "#FFFFFF",
          cursorColor: "#dc1b73",
          barHeight: 100,
          barWidth: 3,
          barRadius: 1,
          responsive: true,
          height: 50,
          width: "100%",
          // normalize: false,
          // partialRender: true,
          url: music.audio_file,
          media: new Audio(),
          peaks: music.beats,
          dragToSeek: true,
          duration: music.duration_seconds,
        });

        // ws.load(music.url);
        ws.on("seeking", (currentTime) => {
          console.log("======seeker", currentTime); // always currentTime = 0, even cursor is not moving after click
        });
```
Is there any help or options to move to specific portion of playing when using custom peaks data?

**Comment by katspaugh** [comment]
You could try the same audio URL but with a regular HTML audio element. Chances are it's something with either the audio file or with your server (e.g. it doesn't support byte range requests).
